src/content/docs/reference/configuration.mdx

```diff
@@ -831,12 +810,48 @@ Whether to enforce collapsing object literals when possible.
 
 > Default: `"preserve"`
 
+### `json.formatter.expand`
+
+Whether to expand arrays and objects literals on multiple lines.
+- `"followSource"`: arrays and objects literals are formatted by following Prettier heuristics
```

Comment:
Might be better to explain the hueristic. I wouldn't expect everyone to know the Prettier behaviour.

```suggestion
- `"followSource"`: arrays and objects literals are formatted on multiple lines if they already were on multiple lines, or if they don't fit on a single line.
```